jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/forumAdministration.js

```diff
@@ -590,24 +606,27 @@ function showGroupCreateDialog(event) {
         }
     });
 
+    if (editMode) fillGroupManagementDialogFields(groupName, groupDescription);
+
     /**
-     * Handles submit request from groupCreateDialog by sending POST request, with params
-     * containing group name and description.
+     * Handles submit request from groupManagementDialog by sending POST or PUT request, with params
+     * containing group information.
      * 
      * @param event
      */
     function sendNewGroup(event) {
         event.preventDefault();
 
         var groupInformation = {};
+        groupInformation.id = groupId;
         groupInformation.name = jDialog.dialog.find('#groupName').val();
         groupInformation.description = jDialog.dialog.find('#groupDescription').val();
 
         jDialog.dialog.find('*').attr('disabled', true);
 
         $.ajax({
-            url: $root + '/group/new',
-            type: 'POST',
+            url: $root + '/group/' + (editMode ? groupId : ''),
```

Comment:
So let's then leave your previous version here and then @Temdegon will update this logic when he refactors the Controllers. Or you can do the refactoring right in this commit since it doesn't relate to the new logic.